backend/src/modules/Announcements/controller/Announcements.controller.spec.js

```diff
@@ -64,6 +64,61 @@ describe('AnnouncementsController |', () => {
     expect(body).toEqual({ message: 'Error message' });
   });
 
+  test('GET /rest-api/borders/:id/announcements | when return board announcements', async () => {
+    // GIVEN
+    const expectedId = uuidv4();
+    const expectedId2 = uuidv4();
```

Comment:
sugeruję jak poniżej, ale też ok :)
```suggestion
    const testId = uuidv4();
    const testId = uuidv4();
```

Reply:
done